src/manager.cpp

```diff
@@ -151,13 +166,107 @@ void Manager::deleteSingleFruVpd(
     logging::logMessage(std::string(i_dbusObjPath));
 }
 
-std::string Manager::getExpandedLocationCode(
-    const sdbusplus::message::object_path& i_dbusObjPath)
+std::string Manager::getExpandedLocationCode(const std::string& i_locationCode,
+                                             uint16_t i_nodeNumber)
 {
-    // Dummy code to supress unused variable warning. To be removed.
-    logging::logMessage(std::string(i_dbusObjPath));
+    (void)i_nodeNumber;
+    std::string o_expandedLocationCode;
+    if (!isValidLocationCode(i_locationCode))
+    {
+        throw types::CommonError::InvalidArgument();
```

Comment:
Doxygen of function does not have information about throw.

Reply:
Updated